common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -93,6 +93,9 @@ public class LocalMSALController extends BaseController {
     @SuppressWarnings(WarningType.rawtype_warning)
     private IAuthorizationStrategy mAuthorizationStrategy = null;
 
+    @SuppressWarnings({WarningType.rawtype_warning, WarningType.unchecked_warning})
```

Comment:
merge this pr back separately (so that it's easier for us to track down the changes in the future)